TLM/TLM/Resources/lang.txt

```diff
@@ -231,4 +231,8 @@ Also_apply_to_left/right_turns_between_one-way_streets Also apply to left & righ
 Scan_for_known_incompatible_mods_on_startup Scan for known incompatible mods on startup
 Ignore_disabled_mods Ignore disabled mods
 Traffic_Manager_detected_incompatible_mods Traffic Manager detected incompatible mods
-Notify_me_if_there_is_an_unexpected_mod_conflict Notify me if there is an unexpected mod conflict
\ No newline at end of file
+Notify_me_if_there_is_an_unexpected_mod_conflict Notify me if there is an unexpected mod conflict
+Speed_limit_unlimited No limit
+Display_speed_limits_mph Display speed limits as mph instead of km/h
```

Comment:
Capitalize MPH & KM/H

Reply:
International abbreviation for miles per hour is MPH and mph, for kilometres per hour is lowercase km/h. So debatable.

Reply:
Yes, debatable. In my opinion, I live in the U.S. currently and I have never once seen it in lowercase (it's always in capital letters). I think in terms of abbreviations it should be capitalized.